flocker_bb/builders/flocker_vagrant.py

```diff
@@ -591,7 +595,7 @@ def getSchedulers():
             if configuration.provider != 'vagrant'
             and configuration.distribution == distribution
         ]
-        if distribution == 'fedora-20':
+        if distribution == TUTORIAL_DISTRIBUTION:
             builders.append('flocker-vagrant-tutorial-box')
```

Comment:
I'd like to rename this to `flocker/vagrant/build/tutorial` while we are changing the builder.